package.json

```diff
@@ -265,9 +347,223 @@
 					"type": "number",
 					"default": 5120,
 					"description": "Max files for searching"
+				},
+				"todohighlight.styles": {
+					"type": "array",
+					"items": {
+						"type": "object",
+						"properties": {
+							"enabled": {
+								"type": "boolean",
+								"default": true,
+								"description": "Enable or disable this style from the UI."
+							},
+							"id": {
+								"type": "number",
+								"minimum": 1,
+								"maximum": 10
+							},
+							"backgroundColor": {
+								"type": "string",
+								"description": "The text background color."
+							},
+							"border": {
+								"type": "string",
+								"description": "The border style for the highlight, as a CSS string."
+							},
+							"color": {
+								"type": "string",
+								"description": "The text color. "
+							},
+							"cursor": {
+								"type": "string",
+								"description": "The style for the cursor shown over the highlight, as a CSS property."
+							},
+							"isWholeLine": {
+								"type": "boolean",
+								"default": false,
+								"description": "If true, then the whole line is highlighted, not just the matching text."
+							},
+							"overviewRulerColor": {
+								"type": "string",
+								"description": "The color of the ruler mark on the scroll bar."
+							}
+						},
+						"required": [
+							"id"
+						]
+					},
+					"default": [
+						{
+							"id": 1,
+							"enabled": true,
+							"backgroundColor": "#ffeb3b"
+						},
+						{
+							"id": 2,
+							"enabled": true,
+							"backgroundColor": "#2196f3"
+						},
+						{
+							"id": 3,
+							"enabled": true,
+							"backgroundColor": "#ff9800"
+						},
+						{
+							"id": 4,
+							"enabled": false,
+							"backgroundColor": "#9c27b0"
+						},
+						{
+							"id": 5,
+							"enabled": false,
+							"backgroundColor": "#2e7d32"
+						},
+						{
+							"id": 6,
+							"enabled": false,
+							"backgroundColor": "#ff5722"
+						},
+						{
+							"id": 7,
+							"enabled": false,
+							"backgroundColor": "#607d8b"
+						},
+						{
+							"id": 8,
+							"enabled": false,
+							"backgroundColor": "#795548"
+						},
+						{
+							"id": 9,
+							"enabled": false,
+							"backgroundColor": "#ffeb3b"
+						},
+						{
+							"id": 10,
+							"enabled": false,
+							"backgroundColor": "#2196f3"
+						}
+					]
```

Comment:
Need to edit the default styles. 
(I didn't write those, I let copilot generate them for me)